src/pages/Search/index.tsx

```diff
@@ -1,12 +1,12 @@
-import { ReactComponent as PreviousIcon } from 'assets/svg/previous.svg';
 import { ReactComponent as LensIcon } from 'assets/svg/lens.svg';
-import { Link, useSearchParams } from 'react-router-dom';
+import { useSearchParams } from 'react-router-dom';
 import { useRef, useState } from 'react';
 import list from './static/data';
 import styles from './Search.module.scss';
 import recommend_text from './static/recommend';
 import cn from '../../utils/ts/classNames';
 import RollingBanner from './components/RollingBanner';
+import PreviousButton from '../../components/PreviousButton/PreviousButton';
```

Comment:
저희 프로젝트는 절대경로 접근을 제공합니다.
```suggestion
import cn from 'utils/ts/classNames';
import RollingBanner from 'components/RollingBanner';
import PreviousButton from 'components/PreviousButton/PreviousButton';
```

Reply:
수정완료했습니다 👍
[137b9a4](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/26/commits/137b9a42886a5a07426075fab5c4791b0180b90a)